etna/search/views.py

```diff
@@ -161,9 +163,11 @@ def catalogue_search(request):
                 Aggregation.CATALOGUE_SOURCE,
                 Aggregation.CLOSURE,
                 Aggregation.COLLECTION,
-                Aggregation.GROUP,
                 Aggregation.LEVEL,
                 Aggregation.TOPIC,
+                # Fetching more groups so that we receive a counts
+                # for any bucket/tab options we might be showing
+                f"{Aggregation.GROUP}:30",
             ],
```

Comment:
Can we have it as a constant like 'AGGREGATION_SIZE' used in catalogue_search_long_filter_chooser than hardcode?

Reply:
I'm not sure that's worth it, personally? I think the comment does a good enough job of explaining what's happening - and using a constant in just this one case and not the others, where we're relying on the default (10)

Reply:
Somewhat agree, but I will leave this to your best judgment :)

Reply:
In this case, I think including 30 directly is slightly clearer.